app/layouts/SiloLayout.tsx

```diff
@@ -68,7 +68,7 @@ export default function SiloLayout() {
             <Metrics16Icon /> Utilization
           </NavLinkItem>
           <NavLinkItem to={pb.siloAccess()}>
-            <Access16Icon /> Access
+            <Access16Icon /> Silo Access
```

Comment:
I kept these renames from the silo access page changes because they're easy wins.